components/camera/fake/camera_test.go

```diff
@@ -7,25 +7,62 @@ import (
 	"go.viam.com/test"
 
 	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/rimage/transform"
 )
 
-func TestFakeCamera(t *testing.T) {
-	camOri := &Camera{Name: "test", Model: fakeModel}
-	cam, err := camera.NewFromReader(context.Background(), camOri, fakeModel, camera.ColorStream)
+func TestFakeCameraHighResolution(t *testing.T) {
+	model, width, height := fakeModel(1280, 720)
+	camOri := &Camera{Name: "test_high", Model: model, Width: width, Height: height}
+	cam, err := camera.NewFromReader(context.Background(), camOri, model, camera.ColorStream)
 	test.That(t, err, test.ShouldBeNil)
+	cameraTest(t, cam, 1280, 720, 812050, model.PinholeCameraIntrinsics, model.Distortion)
+	err = cam.Close(context.Background())
+	test.That(t, err, test.ShouldBeNil)
+}
+
+func TestFakeCameraMedResolution(t *testing.T) {
+	model, width, height := fakeModel(640, 360)
+	camOri := &Camera{Name: "test_high", Model: model, Width: width, Height: height}
+	cam, err := camera.NewFromReader(context.Background(), camOri, model, camera.ColorStream)
+	test.That(t, err, test.ShouldBeNil)
+	cameraTest(t, cam, 640, 360, 203139, model.PinholeCameraIntrinsics, model.Distortion)
+	err = cam.Close(context.Background())
+	test.That(t, err, test.ShouldBeNil)
+}
+
+func TestFakeCameraLowResolution(t *testing.T) {
+	model, width, height := fakeModel(320, 0)
```

Comment:
[nit] I would also test the cases where width is zero and where both are zero.

Reply:
done